responds_to_parent.gemspec

```diff
@@ -13,8 +13,8 @@ Gem::Specification.new do |s|
   s.files = Dir.glob('lib/**/*')
 
   s.add_runtime_dependency('actionpack', '>= 3.2.22', '< 5.0')
-  s.add_runtime_dependency('prototype-rails')
 
+  s.add_development_dependency('prototype-rails')
```

Comment:
This is really the only important change here. There is no need for this to be a runtime dependency.